src/main/java/org/broadinstitute/hellbender/tools/spark/sv/discovery/inference/CpxVariantDetector.java

```diff
@@ -44,7 +44,8 @@
  * paint the full picture we could decide to emit all BND records,
  * but that could be dealt with later.
  */
-public final class CpxVariantDetector {
+public final class CpxVariantDetector implements Serializable {
```

Comment:
Why is this `java.io.Serializable`? I don't see where this class get serialized.. I always get nervous when we mix java serialization with kryo serialization.

Reply:
If it's for some reason necessary because you're serializing the inner class, might be better to bump the inner class to a top-level class.